sdcm/tester.py

```diff
@@ -2986,6 +2991,7 @@ def tearDown(self):
         time.sleep(1)  # Sleep is needed to let final event being saved into files
         self.save_email_data()
         self.argus_collect_gemini_results()
+        self.collect_certs()
```

Comment:
Shouldn't it be called before the line `2988` (`self.clean_resources()`)?

Reply:
Certs are collected from SCT runner, so cleaning cluster resources doesn't affect collecting certs.

Reply:
we might also consider collecting from nodes, to validate we didn't mess-up transferring them to the correct nodes ?

Reply:
Gathering certs from nodes is also interesting, IMHO.
But we can do it later.

Reply:
Added gathering certificates and cqlshrc from nodes (we had an a failure once with the error `configparser.DuplicateOptionError: While reading from '/home/ubuntu/.cassandra/cqlshrc' [line  8]: option 'usercert' in section 'ssl' already exists`, so collecting cqlshrc would help in investigating, if it occurs again)

Reply:
So, probably now it should be moved before the step where DB nodes get cleaned up?

Reply:
But maybe it is better to move the 'collect_certs` method before `clean_resources` one. If it confused once, will confuse again. And rename it to smth like collect_ssl_conf